src/AuthLDAP.php

```diff
@@ -2950,6 +2956,7 @@ public static function connectToServer(
         $deref_options = 0,
         $tls_certfile = "",
         $tls_keyfile = "",
+        $tls_version = "",
```

Comment:
This new parameter should be placed after existing ones, to ensure that existing calls from plugins will still work as expected.